package/boot/uboot-imx6/Makefile

```diff
@@ -48,31 +48,24 @@ define U-Boot/nitrogen6s1g
   NAME:=Nitrogen6x i.MX6Solo 1GB board
 endef
 
-define U-Boot/wandboard_dl
-  NAME:=Wandboard Dual Lite
-  BUILD_DEVICES:=wandboard
-endef
-
-define U-Boot/wandboard_quad
-  NAME:=Wandboard Quad
-  BUILD_DEVICES:=wandboard
-endef
-
-define U-Boot/wandboard_solo
-  NAME:=Wandboard Solo
+define U-Boot/wandboard
+  NAME:=Wandboard Dual Lite/Quad/Solo
   BUILD_DEVICES:=wandboard
 endef
 
 UBOOT_TARGETS := \
-	mx6qsabresd \
+	mx6sabresd \
 	nitrogen6dl \
 	nitrogen6dl2g \
 	nitrogen6q \
 	nitrogen6q2g \
 	nitrogen6s \
 	nitrogen6s1g \
-	wandboard_dl \
-	wandboard_quad \
-	wandboard_solo
+	wandboard
+
+define Build/Compile
+	$(MAKE) -C $(PKG_BUILD_DIR) \
+		CROSS_COMPILE=$(TARGET_CROSS) u-boot.imx
+endef
```

Comment:
Instead of manually defining the compile set please just define that you want to have the u-boot.imx image like this:
UBOOT_MAKE_FLAGS += u-boot.imx